File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/bluefin-tecs-merchant-services-sdk/go/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}

File: rb/utility/make_context.rb
# BluefinTecsMerchantServices SDK utility: make_context
require_relative '../core/context'
module BluefinTecsMerchantServicesUtilities
  MakeContext = ->(ctxmap, basectx) {
    BluefinTecsMerchantServicesContext.new(ctxmap, basectx)
  }
end
